build_js.sh

```diff
@@ -53,7 +53,7 @@ echo "Run:"
 echo "python2 -m SimpleHTTPServer 8000"
 echo "Or:"
 echo "python3 -m http.server"
-echo "Then open in browser:"
-echo "http://0.0.0.0:8000/build_js/utils/viewer/viewer.html?scene=skokloster-castle.glb"
```

Comment:
I removed this reference to viewer.html. I consider it deprecated and it's not a good starting point for JS anymore.